CodeReview/Maru_code_forReview.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+import pandas as pd
+import xarray as xr
+from netCDF4 import Dataset
+
+
+def ferry_data_download(URL):
+	"""Downloads the ADCP data from ferry"""
+	#Inputs:
+	# URL with the location of the ferry NetCDF file
+	#Outputs:
+	# An object containing the link to the ferry data
+	ferry = xr.open_dataset(URL)
```

Comment:
Incorporate a try - except loop here to see if the URL actually connects, and if not, raise an exception.